server_extensions_extra/src/main/resources/batm-extensions.xml

```diff
@@ -72,6 +72,7 @@
                 <cryptocurrency buyonly="true">BCH</cryptocurrency>
                 <cryptocurrency>BTC</cryptocurrency>
                 <cryptocurrency>LTC</cryptocurrency>
+                <cryptocurrency buyonly="true">BTBS</cryptocurrency>
```

Comment:
Are you adding two-way (buy & sell) or just one way (buy only) support for BTBS?
Here you specify "buyonly=true" and in that case you do not need to implement the "PaymentSupport" interface.